source/Playnite/Settings/PlayniteSettings.cs

```diff
@@ -1351,7 +1351,7 @@ public static void SetBootupStateRegistration(bool runOnBootup)
             if (runOnBootup)
             {
                 FileSystem.DeleteFile(shortcutPath);
-                Programs.CreateShortcut(PlaynitePaths.DesktopExecutablePath, "", "", shortcutPath);
+                Programs.CreateShortcut(PlaynitePaths.DesktopExecutablePath, "--hidesplashscreen", "", shortcutPath);

```

Comment:
Create new `CmdLineOptions` object with appropriate settings and then pass the result via `ToString` method instead for direct string.

Reply:
No worries.  Pull request updated.